ci-operator/step-registry/openshift-qe/installer/bm/poweroff/openshift-qe-installer-bm-poweroff-commands.sh

```
@@ -20,16 +20,30 @@ export LAB_CLOUD
 QUADS_INSTANCE=$(cat ${CLUSTER_PROFILE_DIR}/quads_instance_${LAB})
 export QUADS_INSTANCE
 
+if [[ "${OCP_INVENTORY_OVERRIDE}" == "true" ]]; then
+  OCPINV=$(cat ${CLUSTER_PROFILE_DIR}/ocp_inventory_path)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)